free-1.md

```diff
@@ -57,12 +95,24 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think time would be split more towards systems at the beginning, when making an
+MVP. But once we progress past single command=single input, I have been extremely
+conflicted about how to deal with any kind of global state or conditionals, which
+can be very important in some games.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is a project that should be relatively easy to get something working, but new
```

Comment:
This is a good point - ideally your language will be designed to grow and be extensible so it can handle multiple games and types of games.